scripts/staging.sql

```diff
@@ -64,91 +50,86 @@ DROP TABLE IF EXISTS `test_suites`;
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `test_suites_type` varchar(75) DEFAULT NULL,
+  `test_suite_abbrev` varchar(25) NOT NULL,
+  `test_suite` varchar(75) NOT NULL,
   PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
--- Table structure for table `test_automation_flags`
+-- Table structure for table `test_sub_suites`
 --
 
-DROP TABLE IF EXISTS `test_automation_flags`;
+DROP TABLE IF EXISTS `test_sub_suites`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
-CREATE TABLE `test_automation_flags` (
+CREATE TABLE `test_sub_suites` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `flag` varchar(75) DEFAULT NULL,
+  `testrail_id` int(11) NOT NULL,
+  `test_sub_suite_abbrev` varchar(25) NOT NULL,
+  `test_sub_suite` varchar(75) NOT NULL,
   PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 
-
 --
--- Table structure for table `projects`
+-- Table structure for table `test_automation_status`
 --
 
-DROP TABLE IF EXISTS `projects`;
+DROP TABLE IF EXISTS `test_automation_status`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
-CREATE TABLE `projects` (
+CREATE TABLE `test_automation_status` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `project_name` varchar(75) DEFAULT NULL,
+  `testrail_id` int(11) NOT NULL,
+  `status` varchar(75) DEFAULT NULL,
   PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 
-
-
 --
--- Table structure for table `github_test`
+-- Table structure for table `test_automation_coverage`
 --
 
-DROP TABLE IF EXISTS `github_test`;
+DROP TABLE IF EXISTS `test_automation_coverage`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
-CREATE TABLE `github_test` (
+CREATE TABLE `test_automation_coverage` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `open` int(11) NOT NULL,
-  `closed` int(11) NOT NULL,
-  `label` varchar(50) DEFAULT NULL,
-  `project_name` varchar(50) DEFAULT NULL,
-  `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `testrail_id` int(11) NOT NULL,
+  `coverage` varchar(75) DEFAULT NULL,
   PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+
 --
--- Table structure for table `testrail_test_coverage`
+-- Table structure for table `report_test_coverage`
 --
 
-DROP TABLE IF EXISTS `testrail_test_coverage`;
+DROP TABLE IF EXISTS `report_test_coverage`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
-CREATE TABLE `testrail_test_coverage` (
+CREATE TABLE `report_test_coverage` (
```

Comment:
This is the main data table where data will be inserted from Testrail regularly (all other tables are ref tables)